server/__test__/index.test.js

```diff
@@ -1,6 +1,27 @@
-function sum(a, b) {
-  return a + b;
-}
-test("adds 1 + 2 to equal 3", () => {
-  expect(sum(1, 2)).toBe(3);
+const supertest = require('supertest');
+const router = require('./../app');
+

```

Comment:
Zein please add a test for successfully post request 
And also add some test for database functions